gradio/blocks.py

```diff
@@ -66,11 +66,20 @@
 
 
 class Block:
-    def __init__(self, *, render=True, elem_id=None, visible=True, **kwargs):
+    def __init__(
+        self,
+        *,
+        render: bool = True,
+        elem_id: str | None = None,
+        visible: bool = True,
+        root_url: str | None = None,  # URL that is prepended to all file paths
```

Comment:
Should this be at the Blocks level instead? All individual Block(s) are within the same Blocks right?

Reply:
I put it at the `Block` level so that we can handle the case of loading from different Spaces within a single `Blocks`. E.g. 

```py
import gradio as gr

with gr.Blocks() as demo:
    with gr.Tab("Demo 1"):
        gr.Interface.load("spaces/sayakpaul/raindrop-deraining-maxim")
    with gr.Tab("Demo 2"):
        gr.Interface.load("spaces/gradio/test-loading-examples")

demo.launch()
```

Reply:
Makes sense! Thank you for explaining.